mmseg/models/losses/accuracy.py

```diff
@@ -46,10 +46,13 @@ def accuracy(pred, target, topk=1, thresh=None, ignore_index=None):
         correct = correct & (pred_value > thresh).t()
     correct = correct[:, target != ignore_index]
     res = []
+    eps = 1e-6
```

Comment:
Should we use eps in `torch.finfo`?
https://pytorch.org/docs/stable/type_info.html?highlight=finfo#torch.torch.finfo

Reply:
@MeowZheng

Reply:
Please also refer to https://discuss.pytorch.org/t/torch-epsilon/30497.